rogue-thi-app/pages/personal.js

```diff
@@ -34,61 +68,151 @@ export default function Personal () {
+              <span className="text-muted">
+                {grades && missingGrades && grades.length + '/' + (grades.length + missingGrades.length)}{' '}Noten{' '}
+                <FontAwesomeIcon icon={faChevronRight}/>
+              </span>
+            </div>
+            <span className="text-muted">
+              {ects && ects + ' ECTS'}
+              {average && ' · '}
+              {average && '∅ ' + average}
+            </span>
+          </ListGroup.Item>
+
+        </ListGroup>
+
+        <br/>
+
+        <ListGroup>
+
+          {themes.filter(item => item.style.includes(theme[0])).map(item => (
+            <ListGroup.Item action onClick={() => setShowThemeModal(true)} key={item.style}>
+              <div className={styles.interaction_icon}>
+            <span className="text-muted">
+              {item.name}{' '}
+              <FontAwesomeIcon icon={faChevronRight}/>
+            </span>
+              </div>
+              Theme
+            </ListGroup.Item>
           ))}
-          {renderPersonalEntry('E-Mail', 'email')}
-          {renderPersonalEntry('THI E-Mail', 'fhmail')}
-          {renderPersonalEntry('Telefon', null, () => userdata.telefon || 'N/A')}
-          {renderPersonalEntry('Vorname', 'vname')}
-          {renderPersonalEntry('Nachname', 'name')}
-          {renderPersonalEntry('Straße', 'str')}
-          {renderPersonalEntry('Ort', null, () => userdata.plz && userdata.ort && `${userdata.plz} ${userdata.ort}`)}
+
+          <ListGroup.Item action onClick={() => setShowFoodFilterModal(true)}>
+            <div className={styles.interaction_icon}>
+            <span className="text-muted">
+              Essenspräferenzen{' '}
+              <FontAwesomeIcon icon={faChevronRight}/>
+            </span>
+            </div>
+            Einstellungen
+          </ListGroup.Item>
+
         </ListGroup>
 
-        <AppTabbar />
-      </AppBody>
-    </AppContainer>
-  )
+        <br/>
+
+        <ListGroup>
+
+          <ListGroup.Item action onClick={() => window.open('https://www3.primuss.de/cgi-bin/login/index.pl?FH=fhin', '_blank')}>
+            <FontAwesomeIcon icon={faExternalLink} className={styles.interaction_icon}/>
+            Primuss
+          </ListGroup.Item>
+
+          <ListGroup.Item action onClick={() => window.open('https://moodle.thi.de/moodle', '_blank')}>
+            <FontAwesomeIcon icon={faExternalLink} className={styles.interaction_icon}/>
+            Moodle
+          </ListGroup.Item>
+
+          <ListGroup.Item action onClick={() => window.open('https://outlook.thi.de/', '_blank')}>
+            <FontAwesomeIcon icon={faExternalLink} className={styles.interaction_icon}/>
+            E-Mail
+          </ListGroup.Item>
+
+          <ListGroup.Item action onClick={() => window.open('https://mythi.de', '_blank')}>
+            <FontAwesomeIcon icon={faExternalLink} className={styles.interaction_icon}/>
+            MyTHI (Beschäftigten Portal)
```

Comment:
Can we hide this from users that are not employees?

Reply:
Done.